src/dac.rs

```diff
@@ -27,12 +27,12 @@ impl Dac {
     ///
     /// Only the low 12 bits of `data` will be used, the rest is ignored.
     pub fn write_data(&mut self, data: u16) {
-        self.regs.dhr12r1.write(|w| {
+        self.regs.dhr12r1().write(|w| {
             #[allow(unused_unsafe)]
             // SAFETY: Direct write to register for easier sharing between different stm32f3xx svd
             // generated API
             unsafe {
-                w.dacc1dhr().bits(data)
+                w.dacc1dhr().set(data)
```

Comment:
For some reason this seems to work fine for all(or most) but the f373

Reply:
then use `unsafe bits` until someone fix this in `stm32-rs`